frontend/assets/sass/bulma_overrides/layout/_index.scss

```diff
@@ -1,3 +1,3 @@
 // @import "../../node_modules/bulma/sass/layout/footer";
-@import "../../node_modules/bulma/sass/layout/hero";
+@import "hero";
```

Comment:
How does it know where to load this from?

Reply:
Oh, I see, you've copied over the entire hero file and have made customizations to it...